src/lib/nodes/rate-info-service/RateInfoService.ts

```diff
@@ -31,9 +31,19 @@ export class RateInfoService extends Node<AxiosInstance> {
   protected async checkHealth(): Promise<HealthcheckResult> {
     const start = Date.now()
     const response = await this.getAllRates()
+    this.height = this.formatHeight(response.date)
     return {
       ping: Date.now() - start,
-      height: response.date
+      height: this.height
     }
   }
+
+  private formatHeight(height: number) {
+    if (!height) return 0
+    return Number(
+      Math.ceil(height / 1000)
+        .toString()
+        .substring(2)
+    )
+  }
```

Comment:
The function doesn't need to remain as a private method since it's a utility function. I suggest to move it in `src/lib/nodes/rate-info-service/utils.ts` (create the file if it doesn't exist). Also rename the function to `normalizeHeight`.

Reply:
formatHeight(height: number) -> normalizeHeight(timestamp: number)